test/demo/utils/jestExtensions.ts

```diff
@@ -11,25 +19,31 @@ declare global {
              * @param code Assert that the FDC3Error is thrown with a given `code`
              * @param message Optionally, assert that a given error message is returned
              */
-            toThrowFDC3Error<R>(code: string, message?: string | RegExp): R;
+            toThrowFDC3Error(code: string, message?: string | RegExp): R;
 
             /**
              * Used to test that a RemoteChannel represents the expected client-side Channel
              * @param channel An object with optional `id` and `type` properties that if present we expect the channel to match, or the `ChannelId`
              * @param classType The class we expect the channel to be an instance of
              */
             toBeChannel(channel: {id?: ChannelId, type?: string} | ChannelId, classType?: Function): R;
```

Comment:
Not sure I understand the generics here, is `R` always `CustomMatcherResult`, then?

Reply:
Not sure I do either, but built-in matchers will always return the type that gets passed into the the `expect`, and this continues that. I removed the `<R>` in this line as it seemed redundant

Reply:
✅